.prow.sh

```diff
@@ -4,7 +4,8 @@
 
 # # Only these tests make sense for csi-proxy
 : ${CSI_PROW_TESTS:="unit"}
-: ${CSI_PROW_BUILD_PLATFORMS:="windows amd64 .exe"}
+: ${CSI_PROW_BUILD_PLATFORMS:="windows amd64 .exe nanoserver:1809"}
```

Comment:
CSI_PROW_BUILD_PLATFORMS should already have a default, I think we should remove this line and use that default instead